boa/benches/bench_scripts/expression.js

```diff
@@ -1 +0,0 @@
-1 + 1 + 1 + 1 + 1 + 1 / 1 + 1 + 1 * 1 + 1 + 1 + 1;
```

Comment:
The reason for this benchmark was that for very long operations, the parser could overflow or be very slow. Maybe we don't want to keep it, but we should probably open a discusión on useful benchmarks.